gcampus/tools/urls.py

```diff
@@ -15,21 +15,24 @@
 
 from django.urls import path
 
-from gcampus.analysis.apps import GCampusAnalysisAppConfig
-from gcampus.analysis.views import ODConverterOverView, ODConverterDetailView
-
-# Necessary to get the plots to work
-# from gcampus.analysis.dash_apps import Example_Measurement
-
+from gcampus.tools.apps import GCampusToolsAppConfig
+from gcampus.tools.views import (
+    ODConverterOverView,
+    ODConverterDetailView,
+    MeasurementKitOverView,
+    ToolsOverView,
+)
 
 # Turn off black formatting and pylint
 # fmt: off
 # pylint: disable=line-too-long
 urlpatterns = [
-    path("analysis/convert", ODConverterOverView.as_view(), name="od_converter_overview"),
-    path("analysis/convert/<int:pk>/", ODConverterDetailView.as_view(), name="od_converter"),
+    path("tools/", ToolsOverView.as_view(), name="tools"),
+    path("tools/kits/", MeasurementKitOverView.as_view(), name="measurement_kit_overview"),
+    path("tools/kits/<int:pk_kit>/", ODConverterOverView.as_view(), name="od_converter_overview"),
```

Comment:
The convention is to use `-` instead of `_` in url names. So `measurement_kit_overview` would be `measurement-kit-overview`. I was inconsistent with this at the beginning because I did not know better, but now that we are refactoring anyway, we should probably change it.